### PR TITLE
l10n: Change to a capital letter

### PR DESCRIPTION
**Pull request by Valdnet**
Signed-off-by: Valdnet <47037905+Valdnet@users.noreply.github.com>

### CODE REVIEW

**Review by rakekniven — APPROVED** [review]

**Comment by tcitworld** [comment]
Please change the text in https://github.com/nextcloud/server/blob/master/core/js/tests/specs/setupchecksSpec.js test file as well (`doesn\'t` -> `does not`).

**Review by CarlSchwan — APPROVED** [review]

File: core/js/setupchecks.js [hunk 1]
```diff
@@ -655,7 +655,7 @@
 				var xssfields = xhr.getResponseHeader('X-XSS-Protection') ? xhr.getResponseHeader('X-XSS-Protection').split(';').map(function(item) { return item.trim(); }) : [];
 				if (xssfields.length === 0 || xssfields.indexOf('1') === -1 || xssfields.indexOf('mode=block') === -1) {
 					messages.push({
-						msg: t('core', 'The "{header}" HTTP header doesn\'t contain "{expected}". This is a potential security or privacy risk, as it is recommended to adjust this setting accordingly.',
+						msg: t('core', 'The "{header}" HTTP header does not contain "{expected}". This is a potential security or privacy risk, as it is recommended to adjust this setting accordingly.',
```
Comment (CarlSchwan) [inline core/js/setupchecks.js:658]: This needs to be edited in the unit tests too

**Comment by Valdnet** [comment]
Tests fixed 😉.

**Comment by CarlSchwan** [comment]
Not fully yet 

  Expected 'Fourty Two (Invisible)' to equal 'Fourty Two (invisible)'.

**Comment by tcitworld** [comment]
Those ones are in `core/js/tests/specs/systemtags/systemtagsSpec.js` and `core/js/tests/specs/systemtags/systemtagsinputfieldSpec.js`

**Comment by Valdnet** [comment]
@tcitworld @CarlSchwan Thanks for pointing out the test error 😉.

Test fixed.

**Review by tcitworld — APPROVED** [review]

**Comment by CarlSchwan** [comment]
Test failure is unrelated (phpunit/dav)



1) OCA\DAV\Tests\unit\CalDAV\CalDavBackendTest::testSearchPrincipal
--

 END:VCALENDAR'

**Pull request merged**